schema/tsconfig.json

```
@@ -0,0 +1,9 @@
+{
+  "extends": "../tsconfig.json",
+  "references": [],
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "out",
```

I think we currently output to the same directory, so should this line be this?
```suggestion
    "outDir": ".",
```

---

No, currently we don't output at all. Using `.` would actually not work because the output here are `.d.ts` and the `.json` source files, so the output would override the input.

---

Ah OK, is this file required in this directory? It seems odd having a config file and we don't output anything.

---

Yes, because multiple other TypeScript projects import these files, and every imported TS file needs to belong to exactly one project. We actually do output something as of this PR (the .json and .d.ts files)

---

Cool, got it!